src/app/(landing)/@about/page.tsx

```diff
@@ -1,5 +1,51 @@
-function About() {
-  return <div>about</div>;
+import React from 'react';
+import Container from '@/components/Container';
+import Flex from '@/components/Flex';
+import Text from '@/components/Text';
+import styles from './about.module.scss';
+
+function AboutPage() {
+  return (
+    <Container className={styles.frame}>
+      <Flex
+        align="center"
+        direction="column"
+        className={styles.container}
+        gap="2x-big"
+      >
+        <Text
```

Comment:
Overall lets use the right html tags for these text elements (h1, h2, p, etc.)

Reply:
@ivanklchung missed one, then we are good

Reply:
okay, i made h2, and p